src/gui/src/Widgets/LCD.cpp

```diff
@@ -318,15 +318,15 @@ void LCDDisplay::digitClicked()
 const char* LCDSpinBox::__class_name = "LCDSpinBox";
 
 // used in PlayerControl
-LCDSpinBox::LCDSpinBox( QWidget *pParent, int nDigits, LCDSpinBoxType type, int nMin, int nMax )
+LCDSpinBox::LCDSpinBox( QWidget *pParent, LCDDigit::LCDType lcdType, int nDigits, LCDSpinBoxType type, int nMin, int nMax )
```

Comment:
How about making both the minimum and maximum values `float`s? This would allow for both non-integer and negative values. In addition, it would make the intention of disabling the limits more explicit by requiring them to be set to e.g . NaN.

Reply:
Good idea.  It would probably make more sense to use positive and negative infinity than NaN though, since then there wouldn't even need to be a special case when the bounds are disabled.

Reply:
Even better. How about making -/+ infinity the default arguments so people could omit them if they don't require bounds?

Reply:
Done!